### PR TITLE
Feature popup

### PR DESCRIPTION
**Pull request by somarib**
In this project, I added:

>- JavaScript to dynamically create and load the project section
>- A pop-up to load each project when clicked a button of that project

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -126,6 +126,7 @@ To get a local copy up and running, follow these steps.
 8. Create the mobile version first
 9. Convert mobile version into a desktop by applying media querries on the mobile version
 10. Add the mobile menu for mobile devices
+11. Add the pop up for the projects
```
Comment (Alfredbis29) [inline README.md:129]: - [ ] Kudos on using the updated README template :wink:

File: index.html [hunk 2]
```diff
@@ -156,128 +156,14 @@ <h2 class="jobtitle">Software Developer</h2>
```
Comment (Alfredbis29) [inline index.html:156]: - [ ] You did great work on this project but you still need to work on something for your project to look more professional, when running your project we can't see the popup for the work section for both mobile and desktop version, check the screenshot below, kindly fix this :point_down: 
popupnot

File: loadProjectAndPopup.js [hunk 3]
```diff
@@ -0,0 +1,329 @@
+const closeModal = () => {
+  document.querySelector('.modal-section-mbm').innerHTML = '';
+  document.getElementById('mb-modal-hide-show').style.display = 'none';
+};
+
+const projectArray = [
```
Comment (Alfredbis29) [inline loadProjectAndPopup.js:6]: - [ ] Please fix it so that your object contains appear when the project is launched. :wink:

**Review by Alfredbis29 — CHANGES_REQUESTED** [review]
## Hello @somarib ,
 I'm Alfred your reviewer for this PR :man_beard:
 
 Great work working on your project so far! :clap: There are some issues that you still need to work on to go to the next project but you are almost there!
 
  

 ## Highlights
Great job setting up the correct linters :heavy_check_mark:
 Meaningful commit messages :heavy_check_mark:
 
 ##  Required Changes :black_circle: 

 - [ ]  _Check the comments under the review._
 
 ## [Optional] suggestions
 

- [ ] N/A

 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

 Cheers and Happy coding! :partying_face: :clap: :clap:

 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me  in your question so I can receive the notification.

 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: loadProjectAndPopup.js (hunk 3 above)
Comment (somarib) [reply loadProjectAndPopup.js:6]: At my end everything is fine as shown in the screen shot above

File: index.html (hunk 2 above)
Comment (somarib) [reply index.html:156]: image

image

At my end  the project loads well

**Review by mckent05 — APPROVED** [review]
Hi @somarib,

# STATUS: APPROVED ✔️ ✔️  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations and good job! 🎉 
approved


## Highlights
- All linter checks are passing ✔️ 
- Popup works as expected 👏 👏 
- Your PR is professional 👍 👍 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**